vision/classification_and_detection/python/openimages.py

```diff
@@ -241,14 +241,21 @@ def __init__(self, use_inv_map, score_threshold, height, width, dict_format=True
         self.height = height
         self.width = width
         self.dict_format = dict_format
+        import torch
+        self.device = "cuda:0" if torch.cuda.is_available() else "cpu"
 
     def __call__(self, results, ids, expected=None, result_dict=None):
         if self.dict_format:
             # If the output of the model is in dictionary format. This happens
             # for the model ssd-resnext50-pytorch
-            bboxes_ = [e['boxes'] for e in results]
-            labels_ = [e['labels'] for e in results]
-            scores_ = [e['scores'] for e in results]
+            if self.device == "cuda:0":
```

Comment:
I don't think the branch is needed, though. If `e` is already on CPU, then `e.cpu()` is a no-op. So maybe we only need to add `.cpu()` always without the need to add the if-branch?